2019-02-26 15:53:37,750 - invest - ERROR - this is error message eeeeee - [logger.py:23]
2019-02-26 15:53:37,751 - invest - INFO - this is info message iiiiii - [logger.py:24]
2019-02-26 15:53:38,863 - LoginTest - INFO - 开始执行第1条用例 - [test_future_api.py:25]
2019-02-26 15:53:39,680 - LoginTest - INFO - 第1条用例执行结果：PASS - [test_future_api.py:31]
2019-02-26 15:53:48,101 - LoginTest - INFO - 开始执行第2条用例 - [test_future_api.py:25]
2019-02-26 15:53:50,083 - LoginTest - INFO - 第2条用例执行结果：PASS - [test_future_api.py:31]
2019-02-26 15:53:50,214 - LoginTest - INFO - 开始执行第3条用例 - [test_future_api.py:25]
2019-02-26 15:53:51,148 - LoginTest - INFO - 第3条用例执行结果：PASS - [test_future_api.py:31]
2019-02-26 15:53:51,340 - LoginTest - INFO - 开始执行第4条用例 - [test_future_api.py:25]
2019-02-26 15:53:52,278 - LoginTest - INFO - 第4条用例执行结果：PASS - [test_future_api.py:31]
2019-02-26 15:53:52,442 - LoginTest - INFO - 开始执行第5条用例 - [test_future_api.py:25]
2019-02-26 15:53:53,366 - LoginTest - ERROR - 第5条用例执行结果：Failed - [test_future_api.py:34]
